### PR TITLE
tests/xtimer_usleep_until: Print statistics at the end

### PR DESCRIPTION
**Pull request by gebart**
Will print some statistics after the test finishes:

```
...
2016-02-16 08:28:11,859 - INFO # Min/max error: 0/9
2016-02-16 08:28:11,859 - INFO # 
2016-02-16 08:28:11,860 - INFO # Test complete.
```

### CODE REVIEW

File: tests/xtimer_usleep_until/main.c [hunk 1]
```diff
@@ -46,12 +48,20 @@ int main(void)
...
+            max_diff = res[i];
+        }
+        if (res[i] < min_diff) {
+            min_diff = res[i];
+        }
+
         if (res[i] < -1000 || res[i] > 1000) {
             puts("too large difference.\n");
             puts("Test Failed.\n");
             return 1;
```
Comment (A-Paul) [inline tests/xtimer_usleep_until/main.c]: This will skip the output of the elaborate statistic. :-)

File: tests/xtimer_usleep_until/main.c [hunk 2]
```diff
@@ -46,12 +48,20 @@ int main(void)
...
         }
     }
+    printf("\nMin/max error: %" PRId32 "/%" PRId32 "\n", min_diff, max_diff);
```
Comment (A-Paul) [inline tests/xtimer_usleep_until/main.c]: Do the error values have some kind of a unit?

**Comment by A-Paul** [comment]
@gebart, good idea. I like statistics. :)

To print the min/max values in any case you may replace line 57-67 with this:

```
    }

    printf("\nMin/max error: %" PRId32 "/%" PRId32 "\n", min_diff, max_diff);

    if (min_diff < -1000 || max_diff > 1000) {
        puts("too large difference.\n");
        puts("Test Failed.\n");
        return 1;
    }

    printf("\nTest complete.\n");
    return 0;
```

**Comment by gebart** [comment]
@A-Paul 
applied the suggested change and immediately squashed.

**Comment by A-Paul** [comment]
@gebart thanks for applying changes. Did a quick test on *arduino-due' and 'native'. O.K. and ACK.

**Comment by A-Paul** [comment]
Merging.

**Comment by gebart** [comment]
Thanks for a well-done review!

**Pull request merged**